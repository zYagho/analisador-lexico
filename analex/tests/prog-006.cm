int a;
int b[10];

int input(void){
  return;
}

void output(int x){

}

int func(int x, int y){
  int res;
  res = x + y - 2;
  if(res != 0){
    res = -1;
  }
  return(res);
}

int main(void){
  a = input();
  b = input();

  b[0] = a;
  b[1] = func(a,b);

  output(a);
  return(0);
}
