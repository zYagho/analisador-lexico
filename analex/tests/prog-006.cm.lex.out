INT
ID
SEMICOLON
INT
ID
LBRACKETS
NUMBER
RBRACKETS
SEMICOLON
INT
ID
LPAREN
VOID
RPAREN
LBRACES
RETURN
SEMICOLON
RBRACES
VOID
ID
LPAREN
INT
ID
RPAREN
LBRACES
RBRACES
INT
ID
LPAREN
INT
ID
COMMA
INT
ID
RPAREN
LBRACES
INT
ID
SEMICOLON
ID
ATTRIBUTION
ID
PLUS
ID
MINUS
NUMBER
SEMICOLON
IF
LPAREN
ID
DIFFERENT
NUMBER
RPAREN
LBRACES
ID
ATTRIBUTION
NUMBER
SEMICOLON
RBRACES
RETURN
LPAREN
ID
RPAREN
SEMICOLON
RBRACES
INT
ID
LPAREN
VOID
RPAREN
LBRACES
ID
ATTRIBUTION
ID
LPAREN
RPAREN
SEMICOLON
ID
ATTRIBUTION
ID
LPAREN
RPAREN
SEMICOLON
ID
LBRACKETS
NUMBER
RBRACKETS
ATTRIBUTION
ID
SEMICOLON
ID
LBRACKETS
NUMBER
RBRACKETS
ATTRIBUTION
ID
LPAREN
ID
COMMA
ID
RPAREN
SEMICOLON
ID
LPAREN
ID
RPAREN
SEMICOLON
RETURN
LPAREN
NUMBER
RPAREN
SEMICOLON
RBRACES
